lib/theme.js

```diff
@@ -1,91 +1,106 @@
 'use strict';
-import bower from 'bower';
 import when from 'when';
 import Path from 'path';
-import mkdirp from 'mkdirp';
 import _ from 'lodash';
+import fs from 'fs';
 import File from 'fs-extra';
-import del from 'del';
 import elegantSpinner from 'elegant-spinner';
 import logUpdate from 'log-update';
-import Dir from './dir';
 import 'source-map-support/register';
 
 let frame = elegantSpinner();
 
-
 /**
- * The class that installs themes.
+ * The class that locates themes
  * @class  Theme
  */
 class Theme {
   constructor(options) {
-    // Set hidden path
-    let hiddenPath = Path.join(Dir.getHomeDir(), '.doxx');
 
-    // Set template options
+    let resolvedTheme = this.locateTheme(options.theme);
```

Comment:
@cliftonc If you don't mind, could you change that to:
```javascript
let resolved = { theme: this.localeTheme(options.theme) };
```
Just to keep it consistent and for the future because we may never know if we have to use `resolved` for other properties. :)